egs2/TEMPLATE/asr1/asr_streaming.sh

```diff
@@ -0,0 +1,1383 @@
+#!/usr/bin/env bash
```

Comment:
Do we need to include a training part in the streaming process?
@eml914 @hfujihara, what do you think?

I think we only need an inference part and just re-use `asr.sh` to train the streaming model with the stream model config in each recipe.
However, if this split is difficult, we can keep this file.

Reply:
I agree with @sw005320 because it is harder to maintain both the `asr.sh` and `asr_streaming.sh`.  The training part can use `asr.sh`.